### ISSUE TITLE
parse error for "property propName;"

### ISSUE THREAD

**Issue by atuttle** [cflint/CFLint#100]
This is valid code, in a CFC:

    property config;

For example, used in conjunction with `acessors="true"`, DI/1 will autowire in an instance of my configuration service (another CFC).

When I attempt to lint it, I get:

```
$ cflint -file filteringEventRegistrations.cfc
line 6:16 no viable alternative at input 'propertyconfig;'
Stage 2!
line 6:16 no viable alternative at input 'propertyconfig;'
<issues>
  <issue severity="ERROR" id="PARSE_ERROR" message="PARSE_ERROR" category="CFLint" abbrev="PE">
    <location file="/Users/adam/dev/{snip}/filteringEventRegistrations.cfc" fileName="filteringEventRegistrations.cfc" column="16" line="6" message="Unable to parse" variable="">
      <Expression><![CDATA[[@15,128:128=';',<65>,6:16]]]></Expression>
    </location>
    <location file="/Users/adam/dev/{snip}/filteringEventRegistrations.cfc" fileName="filteringEventRegistrations.cfc" column="16" line="6" message="Unable to parse" variable="">
      <Expression><![CDATA[[@15,128:128=';',<65>,6:16]]]></Expression>
    </location>
  </issue>
</issues>
```

**Comment by ryaneberly** [comment]
Fixed in cfparser/cfparser master.

**Comment by ryaneberly** [comment]
fixed in 5.1-snapshot
